common/src/main/java/earth/terrarium/cadmus/Cadmus.java

```diff
@@ -31,7 +32,16 @@ public class Cadmus {
 
     public static int FORCE_LOADED_CHUNK_COUNT;
     public static final TagKey<Block> ALLOWS_CLAIM_INTERACTIONS = TagKey.create(Registries.BLOCK, new ResourceLocation(MOD_ID, "allows_claim_interactions"));
+    public static final TagKey<Item> ALLOWS_CLAIM_PICKUP = TagKey.create(Registries.ITEM, new ResourceLocation(MOD_ID, "allows_claim_pickup"));
     public static final TagKey<EntityType<?>> ALLOWS_CLAIM_INTERACTIONS_ENTITIES = TagKey.create(Registries.ENTITY_TYPE, new ResourceLocation(MOD_ID, "allows_claim_interactions"));
+    public static final TagKey<EntityType<?>> ALLOWS_CLAIM_DAMAGE_ENTITIES = TagKey.create(Registries.ENTITY_TYPE, new ResourceLocation(MOD_ID, "allows_claim_damage"));
+    public static final TagKey<EntityType<?>> CAN_GRIEF_ENTITIES = TagKey.create(Registries.ENTITY_TYPE, new ResourceLocation(MOD_ID, "can_grief"));
```

Comment:
Should be moved to a `ModEntityTypeTags` class to not clutter main. ex. https://github.com/terrarium-earth/Ad-Astra/blob/1.20.x/common/src/main/java/earth/terrarium/adastra/common/tags/ModEntityTypeTags.java